core/src/main/java/org/hibernate/ogm/util/impl/Log.java

```diff
@@ -304,4 +304,7 @@
 
 	@Message(id = 89, value = "%1$s does not support queries on polymorphic entities using TABLE_PER_CLASS inheritance strategy. You should try using SINGLE_TABLE instead. Entities: %2$s")
 	HibernateException queriesOnPolymorphicEntitiesAreNotSupportedWithTablePerClass( String datastore, Collection<String> subclassEntityNames );
+
+	@Message(id = 90, value = "addEntity is not allowed in native queries using projection on root Entity")
```

Comment:
Is it possible to give some additional information related to the query and/or which entity caused the problem?

I'm thinking about a user that it is reading a Log and suddenly see this message without knowing for sure which of the hundred queries caused the problem.

If we don't have the additional info that's OK.

I would prefer if the message were something similar to:
```
Projection and addEntity are  not allowed in the same query. Entity <org.hib.Pippo> with query <select p from ... > 
```